api/openapi-spec/swagger.json

```diff
@@ -46070,35 +46070,6 @@
       "required": true
      }
     ]
-   },
-   "/version/": {
```

Comment:
or if we don't want to limit the tail of the url, we can always use `*` like this example: https://github.com/emicklei/go-restful/blob/master/examples/restful-path-tail.go.